tripal_chado/src/Plugin/TripalBackendPublish/ChadoPublish.php

```diff
@@ -1607,4 +1717,60 @@ public function publish(array $options): array {
     return $this->published_or_updated_entities;
   }
 
+  /**
+   * Unpublishes Tripal entities.
+   *
+   * @param array $options
+   *   Associative array defining what and how to unpublish.
+   *   Required keys are:
+   *     'bundle' - The id of the bundle or entity type
+   *     'datastore' - The id of the TripalStorage plugin
+   *   Optional keys are:
+   *     'job' - A Tripal job object
+   *     'batch_size' - Maximum number of records to publish per batch, defaults to 1000
+   *     'unpublish' - A true value to instead unpublish content, needed to get to this function.
+   *     'orphaned' - Used for unpublish to only unpublish orphaned content, defaults to TRUE.
```

Comment:
```suggestion
   *     'batch_size' - Max number of records per batch, defaults to 1000.
   *     'unpublish' - When true unpublish content (i.e. delete entities)
   *     instead of publishing it. Must be present for this function to
   *     be called.
   *     'orphaned' - When unpublishing, only unpublish orphaned content
   *     which are entities missing their chado record, defaults to TRUE.
```